package.json

```diff
@@ -130,13 +129,14 @@
     "rollup-plugin-visualizer": "^5.6.0",
     "sinon": "^9.0.3",
     "strip-ansi": "^6.0.0",
-    "svgo": "^1.3.2",
+    "svgo": "^3.1.0",
+    "taffydb": "^2.7.3",
```

Comment:
This is required by `@mocha/docdash`. Once #5072 is resolved, we shouldn't need this explicit dep anymore.